clang/lib/IndexDataStore/IndexDataStore.cpp

```diff
@@ -180,15 +185,16 @@ void IndexDataStoreImpl::purgeStaleData() {
 
 
 std::unique_ptr<IndexDataStore>
-IndexDataStore::create(StringRef IndexStorePath, std::string &Error) {
+IndexDataStore::create(StringRef IndexStorePath, PathRemapper Remapper,
+                       std::string &Error) {
   if (!sys::fs::exists(IndexStorePath)) {
     raw_string_ostream OS(Error);
     OS << "index store path does not exist: " << IndexStorePath;
     return nullptr;
   }
 
   return std::unique_ptr<IndexDataStore>(
```

Comment:
make_unique (I do realize this was already here though)

Reply:
This one doesn't work since the constructor is private: `error: calling a private constructor of class 'clang::index::IndexDataStore'`, should I make it public or leave it as is?

Reply:
Argh. That's always annoyed me about make_unique. Just leave it I guess.